mozilla-release/mobile/android/geckoview/build.gradle

```diff
@@ -5,40 +5,6 @@ apply plugin: 'kotlin-android'
 
 apply from: "${topsrcdir}/mobile/android/gradle/product_flavors.gradle"
 
-// This converts MOZ_APP_VERSION into an integer
```

Comment:
Do not remove. See how to comment out blocks in [docs](https://github.com/cliqz-oss/cliqz-android/wiki/Coding-style)